src/binder/bind/bind_ddl.cpp

```diff
@@ -75,6 +77,23 @@ unique_ptr<BoundStatement> Binder::bindDropProperty(const Statement& statement)
     return make_unique<BoundDropProperty>(tableID, propertyID, tableName);
 }
 
+unique_ptr<BoundStatement> Binder::bindAddProperty(const Statement& statement) {
+    auto& addProperty = (AddProperty&)statement;
+    auto tableName = addProperty.getTableName();
+    validateTableExist(catalog, tableName);
+    auto catalogContent = catalog.getReadOnlyVersion();
+    auto tableID = catalogContent->containNodeTable(tableName) ?
```

Comment:
consider using/adding getTableIDFromName(tableName) API directly. Also consider rename getTableIDFromName() to getTableID()

Reply:
Added 
```
    inline table_id_t getTableID(const string& tableName) const {
        return nodeTableNameToIDMap.contains(tableName) ? nodeTableNameToIDMap.at(tableName) :
                                                          relTableNameToIDMap.at(tableName);
    }
```